sdks/typescript/client-typescript-eventsource/README.md

```diff
@@ -1,167 +1,127 @@
+        - `isSet('FEATURE_KEY')` - returns true if a feature is set, otherwise false. If a feature doesn't exist returns false;  
 
-async initializeFeatureHub() {
-    featureHubRepository.addReadynessListener((readyness) => {
-      if (readyness === Readyness.Ready) {
-         // maybe take loading screen off...
-      }
-    });
-
-    featureHubRepository
-      .addPostLoadNewFeatureStateAvailableListener((_) =>
-        { // update the UI, or tell user it needs to happen });
-                                                     
-    // prevent updates to features until we say so
-    featureHubRepository.catchAndReleaseMode = true;
-               
-    // indicate some information about the user and client so we can get the right 
-    // rollout strategies applied to our features
-    featureHubRepository.clientContext.userKey('ideally-unique-id')
-      .country(StrategyAttributeCountryName.NewZealand)
-      .device(StrategyAttributeDeviceName.Browser)
-      .build(); 
-
-    // load the config from the config json file
-    const config = (await globalAxios.request({url: 'featurehub-config.json'})).data as ConfigData;
-
-    // listen for features from the url from the json you just loaded above
-    this.eventSource = new FeatureHubEventSourceClient(config.sdkUrl);
-    this.eventSource.init();
-
-    // this lets you add callback listeners for features. If when you "release" below and this changes, you will
-    // get a callback event
-    // featureHubRepository.getFeatureState('SUBMIT_COLOR_BUTTON').addListener((fs: FeatureStateHolder) => {
-    //   // do something with changed colour
-    // });
-    
-    // add an analytics adapter with a random or known CID
-    // featureHubRepository.addAnalyticCollector(new GoogleAnalyticsCollector('UA-1234', '1234-5678-abcd-1234'));
-  }
 
-// somewhere else in the code, depending on say navigation or a pull to refresh event
-featureHubRepository.release();
-``` 
+   * Get feature value through attached listeners (real-time event-driven feature updates)
 
-### A Mobile device
-
-On a mobile device you will swap out the code above:
+This is when, if the feature changes (either from nothing to something, which happens when we first get the features,
+or an update comes through), a callback is made into your provided function. You can have multiple listeners attached.
 
 ```typescript
-    // listen for features from the url from the json you just loaded above
-    this.eventSource = new FeatureHubEventSourceClient(config.sdkUrl);
-    this.eventSource.init();
+featureHubRepository.feature('FEATURE_KEY').addListener((fs: FeatureStateHolder) => {
+  console.log(fs.getKey(), 'is', fs.getFlag());
+});
 ```
 
-with the following:
+### Connecting to Google Analytics 
+
+To see feature analytics (events) fire in your Google Analytics, you will require to have valid GA Tracking ID, e.g. 'UA-XXXXXXXXX-X'.
+You also need to specify a CID - a customer id this is associate with GA.
```

Comment:
You also need to specify a CID - a customer id **this is associate** with GA. << Not sure what this means?